go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -252,8 +254,9 @@ def handle_subprocess_error(stderr)
               new(go_mod_path, match[1], match[2])
           end
 
+          # We don't know what happened so we raise a generic error
           msg = stderr.lines.last(10).join.strip
-          raise Dependabot::DependencyFileNotParseable.
+          raise Dependabot::DependabotError.
             new(go_mod_path, msg)
```

Comment:
Looks like this only accepts on arg

```suggestion
            new(msg)
```